src/Core/src/Platform/iOS/TextViewExtensions.cs

```diff
@@ -65,6 +65,11 @@ public static void UpdateIsReadOnly(this UITextView textView, IEditor editor)
 			textView.UserInteractionEnabled = !(editor.IsReadOnly || editor.InputTransparent);
 		}
 
+		public static void UpdateIsEnabled(this UITextView textView, IEditor editor)
+		{
+			textView.Editable = editor.IsEnabled;
```

Comment:
Should we also disable the selection (`Selectable` property)?

Reply:
at least on iOS, the selection and edition are handled separately. Not sure on other platforms